README.md

```diff
@@ -341,11 +341,10 @@ return a snapshot of your app's data, and view it in detail to debug your app.
 2. Set snapshots with the following command:
 
 ```
-python3 cli/src/cli.py set_snapshot 2054916c4b46c04e04fffa32781bbd2f index.js:21
+python3 cli/src/cli.py set_snapshot index.js:21 --debuggee-id 2054916c4b46c04e04fffa32781bbd2f
 ```
 
 Where:
-*   `2054916c4b46c04e04fffa32781bbd2f` is the debuggee ID
```

Comment:
I would keep this line so we're clear that this is the debuggee ID

Reply:
Thanks, this has been addressed in #19